components/PlantDetail.js

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledImage = styled(Image)`
+  border: 1px solid black;
+  border-radius: 20px;
+`;
+
+export default function PlantDetail({ plantDetail }) {
+  return (
+    <section>
+      <h2>{plantDetail.name}</h2>
+      <StyledImage
+        alt={plantDetail.name}
+        src={plantDetail.image}
+        width={200}
+        height={200}
+      />
+      <p>botanical name : {plantDetail.botanical_name}</p>
+      <p>water need : {plantDetail.water_need}</p>
+      <p>fertiliser season: {plantDetail.fertiliser_season}</p>
+      <Link href="/">
+        <button>Back</button>
+      </Link>
```

Comment:
The practice of wrapping a `<button>` inside a `<Link>` not the beste choice or approach for several reasons:
- A `<button>` element has a different semantic meaning and purpose compared to an anchor `<a>` element. A `<button>` is meant for **actions** such as submitting a form or toggling a menu, while an <a> tag is meant for navigation.
- Screen readers and other assistive technologies rely on the proper use of semantic HTML to convey information to users. Misusing these elements can lead to confusion or a degraded experience for users relying on these technologies.
- The <Link> component is designed to take advantage of Next.js's navigation features. Wrapping a `<button>` with `<Link> `can lead to unexpected behavior or reduced functionality.

```suggestion
      <Link href="/"> Back to plant list</Link>
```